src/Font.cpp

```diff
@@ -507,7 +507,7 @@ void Font::LoadFontPageSettings( FontPageSettings &cfg, IniFile &ini, const RStr
 
 				wchar_t c;
 				if( sCodepoint.substr(0, 2) == "U+" && IsHexVal(sCodepoint.substr(2)) )
-					sscanf( sCodepoint.substr(2).c_str(), "%x", &c );
+					sscanf( sCodepoint.substr(2).c_str(), "%x", (unsigned int *)&c );
```

Comment:
D'oh C++ style casts totally slipped my mind. I have to use reinterpret_cast instead of static_cast because the pointer types are unrelated. Also you need the pointer because sscanf takes addresses of a variable for the format strings (it's not printf). Running some tests just to make sure, using reinterpret_cast<unsigned int *> gives the same results (but no warning) as it should.